packages/shared/src/contexts/payment/StoreKit.tsx

```diff
@@ -70,6 +81,53 @@ export type PurchaseEvent = {
   detail?: string;
 };
 
+const getApplePlusPricing = (metadata: ProductPricingMetadata[]) => {
+  const response = promisifyEventListener<
```

Comment:
Can just return it here directly

Reply:
Let me check. I remember back then there was something in between hence I didn't directly return.

Reply:
We can return it directly. Should be updated by now.